index.md

```diff
@@ -2,6 +2,48 @@
 linktitle: Welcome
 layout: home
 navbarClass: 'navbar-shrinkable'
+faqs:
```

Comment:
can we put this into the matter for faqs.html so that it's all in one file?

Reply:
That doesn't work because front matter for includes is not processed. 

I'd like to keep only layout things in the `_includes` dir and put all text content in  .md files, or in `_data`, if there's a lot of it.